src/main/resources/edu/cornell/kfs/coa/businessobject/datadictionary/Account.xml

```diff
@@ -260,6 +261,75 @@
        </property>
    </bean>
    
+    <bean id="Account-lookupDefinition"
+          parent="Account-lookupDefinition-parentBean"/>
+    <bean abstract="true"
+          id="Account-lookupDefinition-parentBean"
+          parent="Account-lookupDefinition-base-parentBean"/>
+    <bean abstract="true"
+          id="Account-lookupDefinition-base-parentBean"
```

Comment:
I would not recommend overriding these base Account lookup definition beans. Can you introduce the appropriate customizations through the use of field-overriding beans instead? We should already have one defined for overriding the lookup fields.

Also, cu-kfs has some additional inquiry field overrides in the CG module's `Account.xml` file, so you may need to tweak that if you need to modify the Account inquiry.

Reply:
I am not sure how to do an override with insert after. I can do a bean override with a list merge but that will just add the new fields at the end instead of after the subFundGroupCode. Do you have an example on how to do the insert after with the new lookup definition? Thank you.

Reply:
I am not seeing a change for the inquiry, am I missing it?

Reply:
I don't recall seeing a change to the inquiry, but I wanted to mention the other file containing overrides just to be sure. If there weren't Account Inquiry changes, then that part should be fine.

As for doing field overrides on the new Account lookups, I'll have some steps ready shortly.

Reply:
I haven't found any existing cases where we're performing field overrides on new lookups. The closest I can find is a field override on the new Inquiry/Details in `HigherEducationFunction.xml`, so your override could be configured similarly to that.

Here's how the `FieldOverrideForListElementInsert` beans should be set up for overriding new lookup properties:

1. Set "propertyName" to either "formAttributeDefinitions" (when overriding criteria fields) or "displayAttributeDefinitions" (when overriding result fields).
2. Set "propertyNameForElementCompare" to "name".
3. Set "element" to the Attribute Definition bean of the field marking the insertion point. This can be set up as a "ref" to an existing bean or a child of that bean.
4. Set "insertAfter" to either a single Attribute Definition bean or a list of Attribute Definition beans to insert. These can be set up as "refs" to existing beans or as children of those beans. (If you need the lookup fields to have different settings than their base beans, then use `<bean parent="beanName"/>` instead of `<ref bean="beanName"/>`.)

For example, here's the tentative XML for inserting our custom criteria fields into the new Account lookup's criteria section:

```xml
<bean parent="FieldOverrideForListElementInsert"
      p:propertyName="formAttributeDefinitions"
      p:propertyNameForElementCompare="name"
      p:element-ref="Account-subFundGroupCode">
    <property name="insertAfter">
        <list>
            <ref bean="Account-extension-programCode"/>
            <ref bean="Account-extension-appropriationAccountNumber"/>
            <ref bean="Account-extension-majorReportingCategoryCode"/>
        </list>
    </property>
</bean>
```

Reply:
Thank you Chad!